src/EventHandler.cpp

```diff
@@ -630,7 +661,7 @@ bool EventHandler::shift(KeyModifier mod) const
  */
 bool EventHandler::alt(KeyModifier mod) const
 {
-	return ((mod & KEY_MOD_LALT) || (mod & KEY_MOD_RALT));
+	return KeyModifier::KEY_MOD_NONE != ((mod & KeyModifier::KEY_MOD_LALT) | (mod & KeyModifier::KEY_MOD_RALT));
```

Comment:
Minor point, but this can probably be written as:
```cpp
return KeyModifier::KEY_MOD_NONE != (mod & KEY_MOD_ALT);
```

Edit: Actually, to make it more consistent with the other functions that follow, reverse that order:
```cpp
return (mod & KEY_MOD_ALT) != KeyModifier::KEY_MOD_NONE;
```

Reply:
For long-winded comparisons I tend to put the expected comparison result on the left so it's immediately visible.

Reply:
I do too. Just that the next 3 or 4 functions had it the other way around.

Plus, some programmers prefer to reverse the two, since if the left side is a variable, and you accidentally write `=` instead of `==`, it can compile silently and do the wrong thing. I'm pretty sure we have warnings for that now though. On the other hand, if the left side is an (lvalue) expression, then accidental assignment will give a compiler error.

A similar case is when comparing a variable to a constant. Putting the constant on the left is less prone to typo errors.

In this case, we're comparing an lvalue expression to a constant, so we should be good either way.